ci/ci/templates/index.html

```diff
@@ -76,6 +76,9 @@ <h3>PRs</h3>
             {% else %}
               pending
             {% endif %}
+            {% if pt.is_up_to_date() %}
```

Comment:
Super minor, but seems like the asterisk should indicate out of date PR, no? Like, "success, but see note"

Reply:
yeah, that's what I intended!